_features/lin_advance.md

```diff
@@ -19,11 +19,43 @@ Since the root cause is pressure, `LIN_ADVANCE` de-couples extrusion from the ot
+- This smooth extruder operation and respecting of jerk limits ensures no extruder steps are skipped.
+
+### New K value required
+
+As the unit of K has changed, you have to redo the K calibration procedure. See next chapter for details. While old v1 K values for PLA might be between 30-130, you can now expect K to be around 0.1-2.0.
+
+### LIN_ADVANCE can reduce your print acceleration
+
+In v1, if K was set to a high value which can't be handled by your printer it was loosing steps and / or using all of it's processing power was used to execute extruder steps. In v1.5, this is handled much smarter. `LIN_ADVANCE` will now check if it can execute the advance steps as needed. If the needed extruder speed exceeds the extruder jerk limit, it will reduce the print acceleration for the line printed to a value which keeps the extruder speed within the limit.
+
+While you will most likely not run into this on direct drive printers with filaments like PLA, it will happen most likely on bowden printers as they need higher K values and therefore faster speed adaptions. If this happens to an amount you don't want to accept, you have the following options:
+- Check your extruder jerk setting. If you have the feeling it is set to a very conservative value, try increasing it.
+- Keep the extruder acceleration low. This can be achieved by lowering the layer height or line width for example
+- Keep K as low as possible. Maybe you can shorten the bowden tube?
+
+### A note on bowden printers vs. LIN_ADVANCE
+
+A quite common note during development of `LIN_ADVANCE` was that bowden system (and especialy delta printers) are meant to be faster due to the lower moving mass. So lowering the print acceleration as described above would be an inadequate solution. On the other hand, bowden system need a pressure advance feature the most as they usualy have the most problems with speed changes.
+
+Well, `LIN_ADVANCE` was developed because I (Sebastianv650) wasn't satisfied by the print quality I got out of my direct drive printer. While it's a possible point of view to say lowering acceleration on a bowden printer which is meant to be fast is a bad thing, I see it different: If you would be satisfied with the print quality your bowden setup gives you, you wouldn't be reading about a way to do pressure corrections, isn't it? So maye in this case the high possible top speed is quite useless.. Bowdens are an option to keep moving mass low and therefore allowing higher movement speeds, but don't expect them to give the same precise filament laydown ability than a direct drive one.
+It's like painting a picture: Try to paint with a 1m long brush, grabing the rear end of the handle which is made from rubber. Even if you try to compensate for the wobbly brush tip (which is basicaly what `LIN_ADVANCE` does), it will never be as good as using a usual brush.
```

Comment:
"which is **basicaly** what `LIN_ADVANCE` does" --> "which is **basically** what `LIN_ADVANCE` does"